apps/examples/testcase/le_tc/filesystem/fs_main.c

```diff
@@ -1635,7 +1637,7 @@ static void tc_driver_mtd_config_ops(void)
 	int fd;
 	int ret;
 	struct config_data_s config;
-	char *buf = "test";
+	char buf[8] = "test";
```

Comment:
i think this is better.
const char *buf = "test";

Reply:
as  ioctl cmd  "CFGDIOC_GETCONFIG"  will write this buf, 
if defined it as const, then normally SegmentFault
or will crash in TizenRT.